### PR TITLE
Registered services showing page

### PR DESCRIPTION
**Pull request by midlaj711**
Registered Services page UI completed

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 // import'./App.css';
 function App() {
 
 
   return (
-    <Routes>
-      <Route path="/*" element={<ProviderRoutes/>} />
-    </Routes>
+    
+      <Routes>
```
Comment (JaseelaAlimahamed) [inline src/App.jsx]: no need to give in app.js , make a provider router to setup routes

File: src/routes/ProviderRoutes.jsx [hunk 2]
```diff
@@ -2,14 +2,11 @@ import React from 'react'
 import { Route, Routes, Navigate } from 'react-router-dom'
```
Comment (JaseelaAlimahamed) [inline src/routes/ProviderRoutes.jsx:2]: here , setup the routes not direct in app.js

File: src/App.jsx (hunk 1 above)
Comment (midlaj711) [reply src/App.jsx]: updated the app.js and make providerrouter for setup routes.

**Comment by midlaj711** [comment]
updated the app.js and make provider router for setup routes

**Review by JaseelaAlimahamed — APPROVED** [review]

**Pull request merged**